binaries/apiserver/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	level, err := log.ParseLevel(*logLevelFlag)
 	if err != nil {
 		log.Error(err)
-		return
+		level = log.InfoLevel
```

Comment:
+1.   Also - does the logrus Fatal trigger a call to os.Exit(1) (as the log package does?)   The logrus docs just say it logs a message at the Fatal level doesn't mention os.Exit(1).... the next question is do we want it to exit?